include/exadg/utilities/numbers.h

```diff
@@ -24,18 +24,29 @@
 
 namespace ExaDG
 {
-namespace numbers
+namespace types
 {
-unsigned int const steady_timestep = -1;
+using time_step = unsigned int;
 }
 
+namespace numbers
+{
+types::time_step const invalid_timestep = std::numeric_limits<unsigned int>::max();
+types::time_step const steady_timestep  = std::numeric_limits<unsigned int>::max() - 1;
```

Comment:
can we here use `dealii::numbers::invalid_unsigned_int`?

Reply:
I think we have to make sure that steady_timestep and invalid_unsigned_int is not the same to avoid confusion. Therefore, steady_timestep can not be std::nl::max(). I just added the invalid_timestep to ensure it is clear that the values is occupied but we can also get rid of it.

Reply:
Regarding this discussion: what is in your opinion the default value for `trigger_every_time_steps`, which should be an "invalid" value in my opinion as written in another comment?
For completeness: I realized this before by using `int` instead of `unsigned int` and using `-1` as an invalid default value.

Reply:
trigger_every_time_steps is a number and not a time step. It is set invalid in the constructor. I don't see a problem in this regard.